shared/chat/conversation/messages/account-payment/container.js

```diff
@@ -82,6 +82,10 @@ const mapStateToProps = (state, ownProps: OwnProps) => {
 
 const mapDispatchToProps = (dispatch, ownProps) => ({
   onSend: () => {
+    if (Styles.isMobile) {
```

Comment:
lets just make a dummy component in the route and show a `<Text> TODO` kind of thing